README.md

```diff
@@ -26,18 +26,21 @@ At Gitpod, we've been asked a lot about how we do it. So we thought we might as
 ```bash
 docker run -it --init -p 3000:3000 -v "$(pwd):/home/workspace:cached" gitpod/openvscode-server
 ```
-- Visit the URL printed in your terminal. Just going to port 3000 won't work, because VS Code requires that you provide a uniquely generated security token to prevent unauthorized access.
-
-#### A note about security tokens
-
-Since OpenVSCode Server v1.62, you must access the Web UI using a connection token generated by the server. If that's not what you're looking for, and instead want to generate these tokens by yourself, start the server with `--connection-token YOUR_TOKEN` to force your secret to be used. If you want your development environment to be even more secure, create a plaintext file with the desired token as its contents and provide it to the server with `--connection-secret YOUR_SECRET_FILE`.
+- Visit the URL printed in your terminal. Just going to port 3000 won't work, because VS Code requires that you provide a uniquely generated security token to prevent unauthorized access. You can learn how to change and disable this behavior by reading [A note about security tokens
+](#a-note-about-security-tokens). You will need to add these adjustements to the [`Dockerfile`](https://github.com/gitpod-io/openvscode-releases/blob/HEAD/Dockerfile) and build it.
 
 For those using docker-compose you can set a connection token by using `command`. For example:
 
 ```yaml
 command: --connection-token yourTokenHere
 ```
```

Comment:
For docker compose, if user wants to use token, I think they can use `entrypoint` in the yaml to override the entrypoint from docker image